src/test/test-execute.c

```diff
@@ -611,7 +611,7 @@ int main(int argc, char *argv[]) {
                 return EXIT_TEST_SKIP;
         }
 
-        assert_se(setenv("XDG_RUNTIME_DIR", "/tmp/", 1) == 0);
+        assert_se(setenv("XDG_RUNTIME_DIR", "/tmp", 1) == 0);
```

Comment:
hmm, this stuff really should use setup_fake_runtime_dir() actually like the other tests